cea/demand/thermal_loads.py

```diff
@@ -1186,6 +1186,12 @@ def get_temperatures(locator, prop_HVAC):
     type_cs            T0   (copied from input)
     type_dhw           T1   (copied from input)
     type_ctrl          T1   (copied from input)
+    type_vent          T1   (copied from input)
+    MECH_VENT        True   (copied from input, ventilation system configuration)
```

Comment:
max. 10 characters for reading in column names from excel. see my post in the coding notes channel.
I use upper case because in some of your examples you are using upper case
eg `if MECH_VENT:  do something
elif WIN_VENT: do something else`